main/main.cpp

```diff
@@ -1585,6 +1585,17 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 #endif
 		}
 
+		uint32_t boot_display_time = GLOBAL_DEF("application/boot_splash/display_time_msec", 0);
+		ProjectSettings::get_singleton()->set_custom_property_info("application/boot_splash/display_time_msec",
+				PropertyInfo(Variant::INT,
+						"application/boot_splash/display_time_msec",
+						PROPERTY_HINT_RANGE,
+						"0,100,1,or_greater")); // No negative numbers
+		if (boot_display_time > 0) {
+			// delay_msec() isn't yet available.
```

Comment:
We don't necessarily intend to implement it so I guess the comment is not necessary.

Reply:
The function is implemented, it is simply not available in `main.cpp`, unlike `delay_usec`.

![](https://user-images.githubusercontent.com/47700418/94168777-aaa96880-fe96-11ea-843a-df5c99d0df07.png)

https://github.com/godotengine/godot/blob/b592b74d66f8d728cc512bbde5448d9f4c4e1624/core/bind/core_bind.cpp#L506-L512